tools/block-generator/runner/run.go

```diff
@@ -43,8 +43,11 @@ import (
 	"github.com/algorand/go-algorand/tools/block-generator/util"
 )
 
-//go:embed template/conduit.yml.tmpl
-var conduitConfigTmpl string
+//go:embed template/conduit_pg.yml.tmpl
+var conduitPostgresConfigTmpl string
+
+//go:embed template/conduit_fe.yml.tmpl
```

Comment:
I keep reading this as `conduit_iron`. Could you rename this to `conduit_file_exporter.tmpl`?

Reply:
7b6041cf7e7855ad4041012be7e10d7a5c87a56c